packages/monitor-v2/src/monitor-og/SnapshotVerification.ts

```diff
@@ -66,6 +67,97 @@ interface RulesParameters {
   votingPeriod: number;
 }
 
+// Type guard for MainTransaction.
+const isMainTransaction = (transaction: MainTransaction): transaction is MainTransaction => {
```

Comment:
nit: for these type guards, shouldn't the input be `unknown` rather than the type you're trying to cast it to? Otherwise, this is sort of a no-op from typescript's perspective (proving something that's already type A is type A), right?

Reply:
agree, fixed